helm/README.md

```diff
@@ -10,13 +10,17 @@ Install this chart using:
 
 ```bash
  cd helm/seekret
- helm install seekret --namespace seekret --create-namespace .
+ helm install seekret .
 ```
 
+We create a namespace called seekret-injector. This is hard coded since the certificates are signed on those domains.
```

Comment:
does this work if the namesapce already exists?

Reply:
should not exist. this is a unique namespace, if someone is using the same namespace it is weird
if we are already installed - you'll need to delete-install or to upgrade